### DISCUSSION TITLE
Font size conversion

### DISCUSSION THREAD

**Discussion by EliasA97** [olikraus/u8g2#2013] [General]
Hello Oliver, I'm using this specific font for my project "u8g2_font_mozart_nbp_h_all" and it's dimentions are 5x13.
It has the golden ratio for a 128x64 LCD and it prints the characters very clean and nice. 
Right now I'm making a multilingual menu and I want to include the cyrillic alphabet. The font I'm using doen't contain them but I found another one, the "u8g2_font_glasstown_nbp_t_all" and it has many characters including cyrillic. The problem is that this font's dimentions are 12x16. Is it possible to convert it to 5x13?

**Comment by olikraus** [comment]
> Is it possible to convert it to 5x13

Unfortunately not. All the fonts are manually hand crafted.

**Comment by olikraus** [comment]
Sometimes you might be in luck and a newer version or a different font size is available, but this does not seem to the case here (see for example here: https://fontstruct.com/fontstructions/show/712873/glasstown_nbp)

**Comment by EliasA97** [comment]
I see... Well I found a font on the site you mentioned that includes all the characters I need, so when I make it work, I'll post here for anyone else needing it.
